### ISSUE TITLE
Disaggregate emissions by GFED sectors (total, savannah, boreal, temperate, tropical, peat, agri)

### ISSUE THREAD

**Issue by lambwf** [openclimatedata/global-biomass-burning-emissions#8]
This file is great and I use it frequently. There is one issue: some GFED categories overlap with other data sources (e.g. inventories, EDGAR). Also some categories are relevant for WG3 assessments, others are not. Is it possible to have the long time series disaggregated into total, savannah, boreal, temperate, tropical, peat, and agriculture?

**Comment by rgieseke** [comment]
Glad to hear it's being useful!

Do you mean the VUA data from input4MIPS?

**Comment by lambwf** [comment]
Yes. As an example, I know that the Global Carbon Budget only uses the "Tropical deforestation & degradation" and EQAS "Peat fires" (https://www.geo.vu.nl/~gwerf/GFED/GFED4/tables/GFED4.1s_CO2.txt) components in their projection of direct anthropogenic land use change emissions. To harmonise that approach for historic non-CO2 emissions, one would also need those categories in the files provided here (pre-1997).

**Comment by rgieseke** [comment]
Hm, it's been a while that I worked on this but looking at https://aims2.llnl.gov/search/input4mips/?project=input4MIPs&activeFacets=%7B%22institution_id%22%3A%22VUA%22%7D it seems that categorical data is available.

Examples:
- input4MIPs.CMIP6.CMIP.VUA.VUA-CMIP-BB4CMIP6-1-2.atmos.mon.BC-percentage-SAVA.gn
- input4MIPs.CMIP6.CMIP.VUA.VUA-CMIP-BB4CMIP6-1-2.atmos.mon.C10H16-percentage-PEAT.gn
